doc/faq.md

```diff
@@ -0,0 +1,57 @@
+# FAQ
+
+### Different installation methods
+
+The OpenQuake Engine has at least three installation methods. To choose the one that best fits your needs take a look at the **[installation overview](installing/overview.md)**.
+
+***
+
+### Unsupported operating systems
+
+Binary packages *may* work on Ubuntu derivates and Debian if the dependencies are satisfied; these configurations are known to work:
+- **Ubuntu 14.04** (Trusty) packages work on **Mint Linux 17** and on **Debian 8.0** (Jessie)
+- **Ubuntu 16.04** (Trusty) packages work on **Mint Linux 18** and on **Debian unstable**
+
+These configurations however are not tested by our [continuous integration system](https://ci.openquake.org) and we cannot guarantee on the quality of the results. Use at your own risk.
+
+***
+
+### Celery support
+
+In the OpenQuake Engine 2.0 Celery isn't needed (and not recommended) on a single machine setup; the OpenQuake Engine is able to use all the available CPU cores even without Celery.
+Celery must be enabled on a cluster / multi node setup. To enable it please refer to the multiple nodes installation guidelines for [Ubuntu](FIXME)
+or [RedHat/CentOS/SL](FIXME).
+
+***
+
+### MPI support
+
+MPI is not supported by the OpenQuake Engine. Task distribution across network interconnected nodes is made via *Celery* and *RabbitMQ* as broker. No filesystem sharing is needed between the nodes and data transfer is made on plain TCP/IP connection.
```

Comment:
I would add "diskless workers are supported since the workers never interact with the file system"